nvflare/private/fed/client/client_executor.py

```diff
@@ -117,46 +119,22 @@ def reset_errors(self, job_id):
             job_id: the job_id
         """
 
-    @abstractmethod
-    def process_aux_command(self, shareable: Shareable, job_id):
-        """Processes the aux command.
-
-        Args:
-            shareable: aux message Shareable
-            job_id: the job_id
-        """
-        pass
-
 
 class ProcessExecutor(ClientExecutor):
     """Run the Client executor in a child process."""
 
-    def __init__(self, startup):
+    def __init__(self, client, startup):
         """To init the ProcessExecutor.
 
         Args:
             startup: startup folder
         """
+        self.client = client
```

Comment:
Do we need to pass the FL client object here?
Or the cell is enough?

Reply:
the cell should also have name so client.client_name is not required?

Reply:
Client object has more data accessible. It's good to pass it as a parameter here.